packages/contracts-bedrock/snapshots/storageLayout/L1BlockInterop.json

```diff
@@ -76,11 +76,18 @@
     "slot": "7",
     "type": "uint256"
   },
+  {
+    "bytes": "1",
+    "label": "isIsthmus",
+    "offset": 0,
+    "slot": "8",
+    "type": "bool"
+  },
   {
     "bytes": "64",
     "label": "dependencySet",
     "offset": 0,
-    "slot": "8",
+    "slot": "9",
```

Comment:
This slot changing is ok but just noting that it will wipe the dependency set of the interop devnet when we upgrade, but that is ok

Reply:
OK, thanks for noting it. 

I had assumed that interop devnets would be fresh, not upgraded from previous. Happy to change this if it simplifies ops.

Reply:
We will just make them fresh, its not that big of a deal